services/testing-framework/Dockerfile

```diff
@@ -22,6 +22,12 @@ WORKDIR /go/services/testing-framework
 RUN CGO_ENABLED=0 go test -c -run TestClaudie
 
 FROM alpine
+#Add repository label
+LABEL org.opencontainers.image.source "https://github.com/berops/claudie"
```

Comment:
Just for my info, why did you decide to add all these labels ?

Reply:
The GHCR advised me to use them. Some of them have a practical use, i.e. `org.opencontainers.image.source` automatically links repo in GHCR. Also, it looks a bit more professional IMO :).

Reply:
Thanks :)